operator/v1alpha1/types_image_content_source_policy.go

```diff
@@ -56,18 +61,38 @@ type ImageContentSourcePolicyList struct {
 	Items []ImageContentSourcePolicy `json:"items"`
 }
 
-// RepositoryDigestMirrors holds cluster-wide information about how to handle mirros in the registries config.
-// Note: the mirrors only work when pulling the images that are referenced by their digests.
+// RepositoryDigestMirrors holds cluster-wide information about how to handle mirrors in the registries config.
 type RepositoryDigestMirrors struct {
 	// source is the repository that users refer to, e.g. in image pull specifications.
+	// see https://github.com/containers/image/blob/main/docs/containers-registries.conf.5.md#choosing-a-registry-toml-table for format specifications
+	// i.e. using the following format
+	// host[:port]
+	// host[:port]/namespace[/namespace…]
+	// host[:port]/namespace[/namespace…]/repo
+	// +kubebuilder:validation:Required
+	// +kubebuilder:validation:Pattern=`^(([a-zA-Z0-9]|[a-zA-Z0-9][a-zA-Z0-9\-]*[a-zA-Z0-9])\.)*([A-Za-z0-9]|[A-Za-z0-9][A-Za-z0-9\-]*[A-Za-z0-9])(:[0-9]+)?(\/[^\/]+)*$`
```

Comment:
is this a field with pre-existing data in customer clusters?

Reply:
Adding validation or change the current field is not planned for this PR. These have been removed.